### DISCUSSION TITLE
Why the config-map configured for the ingress controller is getting auto modified randomly ?

### DISCUSSION THREAD

**Discussion by biswanaths** [nginx/kubernetes-ingress#4216] [Q&A]
Hello. 

I am seeing random events generated which shows that the config-map is getting updated. There is no manual update on the configmap. I am also not able to find what was changed. 

What are the triggers for the configmap ? 

Can anybody help me find the trigger or in general point to what to look for ? 

Ingress is hosted ok EKS cluster. 
I did rand controller with debug + v5 log level, but there is no information about the event except that only the configmap was updated. 

Thank you. 

Please let me know if any other information is needed.

**Comment by brianehlert** [comment]
The only writing that the ingress controller performs is events back against objects.
Logs would tell us what is happening.  Normal event logging output of the ingress controller pods is pretty informative in helping all of us understand what is happening.
Pod logs and check the configmap status for events.

I am also assuming that you do not have any pipelines that might be repeatedly writing the configmap, even though the source has not changed.  The ingress controller sets watches for updates on config objects and is notified by the K8s API.

**Comment by jasonwilliams14** [comment]
@biswanaths Are you able to provide us with your current configmap setting and `kubectl get <configmap> -oyaml` output of configmap?
Is it the same configmap since deployment or things added/changed/modified?
Thanks.

**Comment by jasonwilliams14** [comment]
@biswanaths one more item. What version are you running of NGINX Ingress controller?

**Comment by vepatel** [comment]
Hi @biswanaths, can you please provide us with the IC logs and info requested in above comments please?

thanks!
